### DISCUSSION TITLE
离线界面包 2022年9月3日

### DISCUSSION THREAD

**Discussion by shuax** [shuax/MouseInc.Settings#362] [Announcements]
增加OCR接口设置

<hr /><em>This discussion was created from the release <a href='https://github.com/shuax/MouseInc.Settings/releases/tag/2022%E5%B9%B49%E6%9C%883%E6%97%A5'>离线界面包 2022年9月3日</a>.</em>

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/188268904-d9189875-4b31-4f08-8440-490e66df368c.png)
zip直接放进去就行。

**Reply by 6Tom** [reply]
牛蛙牛蛙，飞浆的接口😁

**Reply by andydznj** [reply]
感谢作者，终于修理好了

**Reply by 2018011611** [reply]
OCR还是不行。提示未知错误

**Reply by 2018011611** [reply]
后面发现OCR接口换到飞浆后可以使用

**Reply by mMrWang** [reply]
这玩意咋设置啊，哪位大哥来个教程？

**Reply by 6Tom** [reply]
> 这玩意咋设置啊，哪位大哥来个教程？

。。。快速设置>功能开关>更多设置>OCR接口

**Reply by mMrWang** [reply]
> > 这玩意咋设置啊，哪位大哥来个教程？
> 
> 。。。快速设置>功能开关>更多设置>OCR接口



> Add saved reply
> 
> 
> Attach files by dragging & dropping, selecting or pasting them.[ ](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)Styling with Markdown is supported
> Cancel
> Reply
> Write Preview
> Add heading textAdd bold text, <Ctrl+b>Add italic text, <Ctrl+i>
> Add a quote, <Ctrl+Shift+.>Add code, <Ctrl+e>Add a link, <Ctrl+k>
> Add a bulleted list, <Ctrl+Shift+8>Add a numbered list, <Ctrl+Shift+7>Add a task list, <Ctrl+Shift+l>
> Directly mention a user or team
> Reference an issue, pull request, or discussion
> Add saved reply
> 
> 
> Attach files by dragging & dropping, selecting or pasting them.[ ](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)Styling with Markdown is supported
> Comment
> Remember, contributions to this repository should follow our [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
> Category
> [ 📣 Announcements ](/shuax/MouseInc.Settings/discussions/categories/announcements)
> Labels
> None yet
> 4 participants
> [ <img alt="@shuax" width="24" height="24" src="https://avatars.githubusercontent.com/u/6940583?s=48&amp;v=4"> ](/shuax) [ <img alt="@mMrWang" width="24" height="24" src="https://avatars.githubusercontent.com/u/76983769?s=48&amp;v=4"> ](/mMrWang) [ <img alt="@6Tom" width="24" height="24" src="https://avatars.githubusercontent.com/u/88385016?s=48&amp;v=4"> ](/6Tom) [ <img alt="@andydznj" width="24" height="24" src="https://avatars.githubusercontent.com/u/92424490?s=48&amp;v=4"> ](/andydznj)
> Notifications
> Unsubscribe
> You’re receiving notifications because you commented.



> > 这玩意咋设置啊，哪位大哥来个教程？
> 
> 。。。快速设置>功能开关>更多设置>OCR接口

太感谢了！终于好用了！
